webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -15,45 +15,55 @@
         :posting-news="postingNews"
         :news-id="newsId"
         @post-article="postNews" />
-      <div class="newsComposerActions">
-        <div class="newsFormButtons">
-          <div class="newsFormLeftActions">
-            <img src="/news/images/news.png">
-            <span class="newsFormTitle">{{ newsFormTitle }}</span>
+      <div class="newsComposerToolbar">
+        <div class="d-flex flex-row py-2">
+          <v-avatar
+            height="40"
+            min-height="40"
+            width="40"
+            min-width="40"
+            max-width="40"
+            size="40"
+            class="mx-3 my-auto spaceAvatar space-avatar-header">
```

Comment:
why using spaceAvatar space-avatar-header css classes here, can we define another css classes even if we have the same definition ?

Reply:
i don't use this classes i will remove it.